### DISCUSSION TITLE
Problem with nuget config file in working directory of creating classlib by dotnet

### DISCUSSION THREAD

**Discussion by DeafMan1983** [CosmosOS/Cosmos#2576] [General]
Hello everyone, 

I have problem with nuget config file. I have typed and let make compilation with Cosmos on Ubuntu 22.04 but they have built successfully. Nor errors, it looks fine. Yay....

And I copy nupkgs to creating directory packages in classlib's directory Example

1. `dotnet new classlib -f net6.0 -o RabbitOS`
2. `cd RabbitOS`
3. Create nuget.config with gedit -> `gedit nuget.config`
Put XML structure:
```
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <packageSources>
    <add key="packages" value="./packages" />
  </packageSources>
</configuration>
```
Save and close!
4. create directory `mkdir packages`
5. Copy from Cosmos compiled nupkgs files to `./packages` . means current directory is RabbitOS
6. `dotnet restore` and `dotnet add package Cosmos.System2 --prerelease` then it got error. But why does it throw error if I already set up Nuget.config in directory RabbitOS
![image](https://user-images.githubusercontent.com/57066679/218265397-2427c528-667b-48d7-89dc-5a47bc5665d5.png)
And my output said error but it doesn't recognize to catch nupkgs but where is IL2CPP.API ??? I already `make` and `sudo make install` to `/opt/cosmos`

I show you proof because I already copied from Cosmos built nupkgs to ./packages - Tell I show you
![image](https://user-images.githubusercontent.com/57066679/218265506-9c7de01f-bcf5-49e6-9a63-a00e3548606b.png)
![image](https://user-images.githubusercontent.com/57066679/218265563-5408c8da-c635-4c39-b43d-e060e240ab3b.png)

Why does it say `incompatible` but I use correct version of dotnet 6.0.4 and 7.0.2 on Ubuntu 22.04?

I hope you understand me? Sorry my bad English!

**Comment by zarlo** [comment]
you need to run ```make nuget-install```
